src/test/java/com/feelmycode/parabole/controller/CouponControllerTest.java

```diff
@@ -0,0 +1,363 @@
+@RunWith(SpringJUnit4ClassRunner.class)
+public class CouponControllerTest {
+
+    String outputDirectory = "./src/docs/asciidoc/snippets";
+
+    @LocalServerPort
+    int port;
+
+    @Rule
+    public JUnitRestDocumentation restDocumentation = new JUnitRestDocumentation(outputDirectory);
+
+    private RequestSpecification spec;
+    private TokenProvider tokenProvider = new TokenProvider();
+
+    @Autowired
+    private CouponService couponService;
+    @Autowired
+    private CouponRepository couponRepository;
+    @Autowired
+    private UserRepository userRepository;
+
+    @Before
+    public void setUp() {
+        RestAssured.port = port;
+        this.spec = new RequestSpecBuilder().addFilter(
+                documentationConfiguration(this.restDocumentation))
+            .build();
+    }
+
+    @Test
+    @DisplayName("유저 본인이 보유한 쿠폰 목록 조회 : /coupon/list 에 유저 계정으로 테스트")
+    public void couponListUser() {
+
+        Long userId = 39L;
+        String userToken = this.tokenProvider.create(
+        User.builder().id(userId).email("r").password("r").nickname("r").build());    // 유저 계정
+
+        // When
+        Response resp = given(this.spec)
+            .header("authorization", "Bearer " + userToken)
+            .accept(ContentType.JSON)
+            .contentType(ContentType.JSON)
+            .filter(document("coupon-list-user",
+                    preprocessRequest(modifyUris()
+                            .scheme("https")
+                            .host("parabole.com"),
+                        prettyPrint()),
+                    preprocessResponse(prettyPrint()),
+                    responseFields(
+                        fieldWithPath("success").type(JsonFieldType.BOOLEAN).description("성공여부"),
+                        fieldWithPath("message").type(JsonFieldType.STRING).description("메세지"),
+                        fieldWithPath("data").type(JsonFieldType.OBJECT).description("응답 정보"),
+                        fieldWithPath("data.content").type(JsonFieldType.ARRAY).description("쿠폰 목록"),
+                        fieldWithPath("data.content.[].name").type(JsonFieldType.STRING)
+                            .description("쿠폰명").optional(),
+                        fieldWithPath("data.content.[].serialNo").type(JsonFieldType.STRING)
+                            .description("쿠폰 일련번호").optional(),
+                        fieldWithPath("data.content.[].sellerName").type(JsonFieldType.STRING)
+                            .description("셀러 이름").optional(),
+                        fieldWithPath("data.content.[].type").type(JsonFieldType.STRING)
+                            .description("쿠폰 유형").optional(),
+                        fieldWithPath("data.content.[].discountValue").type(JsonFieldType.NUMBER)
+                            .description("할인 값(비율/금액)").optional(),
+                        fieldWithPath("data.content.[].useState").type(JsonFieldType.STRING)
+                            .description("사용 여부").optional(),
```

Comment:
상태의 경우 어떤게 있는지 설명에 써주세요

아래는 예시이고 알맞게 작성해주세요
```suggestion
                        fieldWithPath("data.content.[].useState").type(JsonFieldType.STRING)
                            .description("사용 여부 (사용안함:NOTUSED / 사용함: USED)").optional(),
```